src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlClientEventSource.cs

```diff
@@ -145,7 +145,7 @@ internal void TraceEvent(string message)
         {
             if (Log.IsTraceEnabled())
             {
-                Trace(string.Format(message, args0, args1));
+                Trace(string.Format(message, args0.ToString(), args1.ToString()));
```

Comment:
Can we check for null arguments here? We're unsafely converting to string, which might be risky if values are null here.

Reply:
Also in NetCore project.

Reply:
Fixed.